common/api/imodeljs-common.api.md

```diff
@@ -2136,7 +2142,7 @@ export class EcefLocation implements EcefLocationProps {
 
 // @public
 export interface EcefLocationProps {
-    readonly cartographicOrigin?: LatLongAndHeight;
+    readonly cartographicOrigin?: CartographicProps;

```

Comment:
`LatAndLong` was undocumented - did it use radians or degrees? If radians, this will break things. Note EcefLocationProps is a persistence type.

Reply:
@pmconne 
I see what you mean.  It sounds like the second move I made toward AngleProps was not a good one.
The previous cut of code changes had lat/long in CartographicProps by default using radians unless isDegrees was specified as true (default undefined.)
I'm thinking I should roll back to those changes.
FYI @mgooding